raiden/network/transport/matrix.py

```diff
@@ -575,7 +575,7 @@ def _send_queued_messages(
                 message = _event_to_message(event, node_address)
                 self._raiden_service.sign(message)
                 self.start_health_check(address)
-                self.send_async(address, queue_identifier, message)
+                self.send_async(queue_identifier, message)
```

Comment:
was this working before 0o?

Reply:
Issue here: https://github.com/raiden-network/raiden/issues/2141

Reply:
what about these? https://github.com/raiden-network/raiden/blob/dde8d789a9999f9d407675dd4394b6cbf24566e1/raiden/tests/integration/test_recovery.py